src/olympia/lib/settings_base.py

```diff
@@ -1515,6 +1515,8 @@ def _scrub_sensitive_data_recursively(data, name=None):
         event = _scrub_sensitive_data_recursively(event)
     except Exception:
         pass
+    if 'ip_address' in event.get('payload', {}).get('user', {}):
+        event['payload']['user'].pop('ip_address')
```

Comment:
this wouldn't strip out the other ip addresses fields - `'remote_addr', 'remoteaddresschain', 'x-forwarded-for'` - so I guess we're okay with that?